src/query/storages/common/cache-manager/src/cache_manager.rs

```diff
@@ -38,13 +46,32 @@ pub struct CacheManager {
     bloom_index_filter_cache: Option<BloomIndexFilterCache>,
     bloom_index_meta_cache: Option<BloomIndexMetaCache>,
     file_meta_data_cache: Option<FileMetaDataCache>,
+    table_data_cache: Option<TableDataCache>,
+    table_column_array_cache: Option<ColumnArrayCache>,
 }
 
 impl CacheManager {
     /// Initialize the caches according to the relevant configurations.
-    ///
-    /// For convenience, ids of cluster and tenant are also kept
     pub fn init(config: &QueryConfig) -> Result<()> {
+        // setup table data cache
+        let table_data_cache = if config.table_data_cache_enabled {
+            Self::new_block_data_cache(
+                &config.table_disk_cache_root,
+                config.table_data_cache_population_queue_size,
+                config.table_disk_cache_mb_size * 1024 * 1024,
```

Comment:
`1024 * 1024` -> `* 1024 `?

Reply:
setting name has been changed, and take bytes as unit of size